### PR TITLE
(PC-12409) fix(EAC): fix activity and school types display

### PR DESCRIPTION
**Pull request by yorickeando**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12409

## Checklist

I have:

- [ ] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [x] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/identityCheck/pages/profile/__test__/Status.test.tsx [hunk 1]
```diff
@@ -20,7 +24,12 @@ jest.mock('features/identityCheck/useIdentityCheckNavigation', () => ({
   }),
 }))
 
+const mockUseIsUserUnderage = useIsUserUnderage as jest.Mock
+
 describe('<Status/>', () => {
+  beforeEach(jest.clearAllMocks)
+  afterAll(jest.clearAllMocks)
```
Comment (anoukhello) [inline src/features/identityCheck/pages/profile/__test__/Status.test.tsx:31]: C'est pas redondant les 2 en même temps ?

File: src/features/identityCheck/pages/profile/__test__/Status.test.tsx [hunk 2]
```diff
@@ -31,14 +31,21 @@ describe('<Status/>', () => {
...
   it('should render correctly', () => {
+    mockUseIsUserUnderage.mockReturnValueOnce(true)
+    const renderAPI = render(<Status />)
+    expect(renderAPI).toMatchSnapshot()
+  })
+  // TODO PC-12410 : déléguer la responsabilité au back de faire cette filtration
+  it('should render with no Collégien status if user is over 18', () => {
+    mockUseIsUserUnderage.mockReturnValueOnce(false)
     const renderAPI = render(<Status />)
     expect(renderAPI).toMatchSnapshot()
```
Comment (anoukhello) [inline src/features/identityCheck/pages/profile/__test__/Status.test.tsx:42]: On pourrait regarder que "Collégien" n'est pas présent plutôt que générer 2000 lignes de snapshot pour ça,
ou alors utiliser `toMatchDiffSnapshot`

**Review by anoukhello — APPROVED** [review]

File: src/features/identityCheck/pages/profile/__test__/Status.test.tsx (hunk 1 above)
Comment (yorickeando) [reply src/features/identityCheck/pages/profile/__test__/Status.test.tsx:31]: Mmm peut-être bien, j'ai volé ça quelque part. J'en supprime un

File: src/features/identityCheck/pages/profile/__test__/Status.test.tsx (hunk 2 above)
Comment (yorickeando) [reply src/features/identityCheck/pages/profile/__test__/Status.test.tsx]: yes bien vu

**Pull request merged**